include/rocksdb/listener.h

```diff
@@ -180,7 +179,8 @@ class EventListener {
   // @param ci a reference to a CompactionJobInfo struct. 'ci' is released
   //  after this function is returned, and must be copied if it is needed
   //  outside of this function.
-  virtual void OnCompactionCompleted(DB *db, const CompactionJobInfo& ci) {}
+  virtual void OnCompactionCompleted(DB* /*db*/,
+      const CompactionJobInfo& /*ci*/) {}
```

Comment:
Sometimes type names are obvious and adding a comment is more confusing. I would not add comments when it is more confusing to have the parameter. Those *Info parameters probably won't need comments. Other than that, the change looks good to me.